coriolisclient/tests/cli/test_formatter.py

```diff
@@ -0,0 +1,145 @@
+
+
+class TestEntityFormattter(formatter.EntityFormatter):
+
+    def __init__(self):
+        self.columns = [
+            "column_1",
+            "column_2"
+        ]
+
+    def _get_formatted_data(self, obj):
+        return obj
+
+
+@ddt.ddt
+class TestEntityFormattterTestCase(test_base.CoriolisBaseTestCase):
+    """Test suite for the Coriolis Entity Formatter."""
+
+    def setUp(self):
+        super(TestEntityFormattterTestCase, self).setUp()
+        self.format = TestEntityFormattter()
+
+    def test_get_sorted_list(self):
+        obj_list = ["obj2", "obj1"]
+
+        result = self.format._get_sorted_list(obj_list)
+
+        self.assertEqual(
+            obj_list,
+            result
+        )
+
+    @mock.patch.object(formatter.EntityFormatter, '_get_generic_columns')
+    def test_list_objects(
+        self,
+        mock_get_generic_columns,
+    ):
+        obj_list = ["obj2", "obj1"]
+        mock_get_generic_columns.return_value = mock.sentinel.columns
+
+        result = self.format.list_objects(obj_list)
+
+        self.assertEqual(
+            (
+                mock.sentinel.columns,
+                ["obj2", "obj1"]
+            ),
+            (
+                result[0],
+                list(result[1])
+            )
+        )
+
+    def test_get_generic_data(self):
+        obj = mock.Mock()
+
+        result = self.format._get_generic_data(obj)
+
+        self.assertEqual(
+            obj,
+            result
+        )
+
+    def test_get_generic_columns(self):
+        result = self.format._get_generic_columns()
+
+        self.assertEqual(
+            ["column_1", "column_2"],
+            result
+        )
+
+    @mock.patch.object(formatter.EntityFormatter, '_get_generic_columns')
+    def test_get_formatted_entity(self, mock_get_generic_columns):
+        obj = mock.Mock()
+        mock_get_generic_columns.return_value = mock.sentinel.columns
+
+        result = self.format.get_formatted_entity(obj)
+
+        self.assertEqual(
+            (["column_1", "column_2"], obj),
+            result
+        )
+
+    @ddt.data(
+        {
+            "current_value": 3,
+            "max_value": 9,
+            "percent_format": "{:.0f}%",
+            "expected_result": "33%"
+        },
+        {
+            "current_value": 3,
+            "max_value": 9,
+            "percent_format": "{:.2f}%",
+            "expected_result": "33.33%"
+        },
+        {
+            "current_value": 0,
+            "max_value": 9,
+            "percent_format": "{:.0f}%",
+            "expected_result": None
+        },
+        {
+            "current_value": 3,
+            "max_value": None,
+            "percent_format": "{:.0f}%",
+            "expected_result": None
+        }
+    )
+    def test_get_percent_string(self, data):
+        result = self.format._get_percent_string(
+            data["current_value"],
+            data["max_value"],
+            data["percent_format"]
```

Comment:
Please add keyword to arg: 
```suggestion
            percent_format=data["percent_format"]
```